WalletWasabi.Fluent/App.axaml.cs

```diff
@@ -70,9 +68,12 @@ public override void OnFrameworkInitializationCompleted()
 				RxApp.MainThreadScheduler.Schedule(
 					async () =>
 					{
-						await _backendInitialiseAsync();
+						if (_backendInitialiseAsync is { })
```

Comment:
If `_backendInitialiseAsync` is null that is OK and we can bypass it?

Reply:
In this case it is valid, because when it runs in the Designer (previewer in the IDE) then this property will be null.

Reply:
in this case iv added suppression, because yes it can be null (in designer mode) but this line doesn't get hit in designer mode. So IDE preview will work, and any other time its guaranteed not to be null.

Im usually against suppressing with !..

everywhere else the null checks are justified. most of the time its saying `are we the type of application that has a clipboard` for any avalonia application, the answer can be yes and no, but for wasabi right now its always yes!

The null checks are needed to get rid of compiler warning, and justified because in future we might run somewhere we dont have clipboard.